### PR TITLE
fixes

### PR DESCRIPTION
**Pull request by return2monki**
Few text fixes, great tutorial though really comprehensive!

Some things that could be added in part 1 would be explaining what ds stands for (data segment according to google?) and what the local symbol toggle does as the bgb docs don't mention it.

### CODE REVIEW

**Review by avivace — APPROVED** [review]

File: src/part1/header.md [hunk 1]
```diff
@@ -148,7 +148,7 @@ So, we prevent disaster like this:
 The `ds` fills all bytes from $103 to $14F (inclusive) with the value $00.
 Since different pieces of code and/or data cannot overlap, this ensures that the header's memory range can safely be overwritten by RGBFIX, and that nothing else accidentally gets steamrolled instead.
 
-It may not be obvious how the `ds` fills is told to fill this specific memory range.
+It may not be obvious how `ds` is told to fill this specific memory range.
```
Comment (ISSOtm) [inline src/part1/header.md:151]:
```suggestion
It may not be obvious how this `ds` ends up filling that specific memory range.
```

**Review by ISSOtm — COMMENTED** [review]
I think this was my intent. What do you think?

**Comment by ISSOtm** [comment]
`ds` stands for "define space", actually. The name is not very clear, so I'm not sure it would help to introduce it? OTOH, memorising two letters by heart is, eh.

File: src/part1/header.md (hunk 1 above)
Comment (return2monki) [reply src/part1/header.md:151]: Either is fine, but I just recommend briefly mentioning what it stands for just so people don't have to think about what it means. Not necessarily to memorize it.

File: src/part1/header.md [hunk 2]
```diff
@@ -148,7 +148,7 @@ So, we prevent disaster like this:
 The `ds` fills all bytes from $103 to $14F (inclusive) with the value $00.
```
Comment (ISSOtm) [inline src/part1/header.md:148]:
```suggestion
The directive `ds` stands for "define space", and allows filling a range of memory.
This specific line fills all bytes from $103 to $14F (inclusive) with the value $00.
```

Comment (return2monki) [reply src/part1/header.md:148]: Great

Comment (ISSOtm) [reply src/part1/header.md:148]: Where are they mentioned?

Comment (return2monki) [reply src/part1/header.md:148]: Tracing>setup at the screenshot

Comment (ISSOtm) [reply src/part1/header.md:148]: They're intentionally not explained yet, they'll be brought up somewhere during Part I or Part II.

Comment (return2monki) [reply src/part1/header.md:148]: Alright, you did mention everything that would be explained in part 2. Just not this one.

**Comment by return2monki** [comment]
You can commit to this pr right? Regardless I just added those lines and it should be ready now.

**Pull request merged**